pkg/addons/adminconsole/adminconsole.go

```diff
@@ -38,7 +38,7 @@ const (
 )
 
 var (
-	//go:embed static/values.yaml
+	//go:embed static/values.tpl.yaml
```

Comment:
is there some reason admin console doesnt have architecture images? i dont see any changes to the tpl file here. what am i missing?

Reply:
it uses the `ImageString` template function which has been updated in this pr to account for arch.